src/System.Runtime.WindowsRuntime/src/System/Windows/Rect.cs

```diff
@@ -51,9 +51,9 @@ public struct Rect : IFormattable
                     double height)
         {
             if (width < 0)
-                throw new ArgumentException(nameof(width));
+                throw new ArgumentException(SR.Argument_NeedNonNegNum, nameof(width));
```

Comment:
I'm pretty certain this (and in `Size`) should be `ArgumentOutOfRangeException`, with the equivalent, existing resource, but wanted to check first.